test/unit/participant_test.rb

```diff
@@ -45,8 +45,16 @@ class ParticipantTest < ActiveSupport::TestCase
 
   context "With a proper context, " do
     setup do
-      @participant = FactoryGirl.create(:participant, :phone_number => 1234567890)
+      @participant = FactoryGirl.create(:participant, :phone_number => 1234567890, :andrewid => "agoradia", :cached_name => "Akshay Goradia",)
```

Comment:
I'd prefer to use fake info like "John Smith" etc. Otherwise people are going to constantly change these tests.